src/server/js/resources/bookings/model.js

```diff
@@ -0,0 +1,47 @@
+import mongoose from 'mongoose';
+import { values } from 'lodash';
+import moment from 'moment';
+
+const BOOKING_STATUSES = {
+    APPROVED: 'APPROVED',
+    DENIED: 'DENIED',
+    PENDING: 'PENDING',
+};
```

Comment:
what's a good place to keep these kinds of constants? this will be shared with the frontend also, so maybe
```
src/
  |--client/
  |--server/
  |--constants.js (???)
```

Reply:
how about a common/ or shared/ directory at that level